main.py

```diff
@@ -1,7 +1,32 @@
 import init_django_orm  # noqa: F401
+from db.models import Genre, Actor
 
 from django.db.models import QuerySet
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    for item in genres:
+        Genre.objects.create(name=item)
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+    for name, surname in actors:
+        Actor.objects.create(first_name=name, last_name=surname)
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    Actor.objects.filter(last_name="Klooney").update(last_name="Clooney")
+    Actor.objects.filter(
+        first_name="Kianu",
+        last_name="Reaves"
+    ).update(
+        first_name="Keanu",
+        last_name="Reeves"
+    )
+    Genre.objects.filter(name="Action").delete()
+    Actor.objects.filter(first_name="Scarlett").delete()
```

Comment:
add some blank lines here, for better code style
also, what if we will have another actor with the last name Klooney?
according to the task, u should cahnge a last name for George 
![image](https://github.com/mate-academy/py-genres-and-actors/assets/103201010/a41bbf14-8dd0-4712-b554-d315797bfe5b)